src/app/admin/create-vc-template/create-vc-template.component.html

```diff
@@ -23,7 +24,7 @@
             <div class="thmbCol">
                 <div class="pt-c">
                   <a href="/add-template/{{usecase}}/{{entityName}}" class="btn submit-button pt-c">
-                     <i class="fa fa-plus text-secondary-color pt-c" aria-hidden="true"></i>
+                     <i class="fa fa-plus color-sea-green pt-c" aria-hidden="true"></i>
```

Comment:
@chaitrali-r here you have rename the class name ? color-sea-green ?  is this secondary text color ?

Reply:
Yes, i'll change the class name